.travis.yml

```diff
@@ -10,7 +10,8 @@ before_install:
   - npm install -g grunt@0.4.5
   - npm install -g grunt-cli
 install: npm install
-scripts:
+script:
+  - grunt
```

Comment:
> You need to add {scripts: { test: 'grunt' } } to the package.json

Like in https://github.com/feedhenry-raincatcher/raincatcher-mediator/pull/6/commits/801829f94b834de326c26176c68e482b74bb8276 right?